react/react1/week3/to-do-list-3/src/components/to-do-list.jsx

```diff
@@ -0,0 +1,300 @@
+import { useState } from "react";
+import { useEffect } from "react";
+import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+import BorderBox from "./border-box";
```

Comment:
I see you have done the BorderBox component in a seperate file and then you have a component called TaskToDo in the same file. In general it is okay to have multiple components in the same file. However, i am a bit opiniated on this as i don't like having multiple components in the same file unless they are extremely small like the BorderBox component which is like 3 lines of code. The reason for this is maintainability. I think it quickly gets hard to maintain if you have multiple components in the same file, unless they are incredibly small and maybe just a div with some styling.